apps/displays/Oculus_Rift_DK2.json

```diff
@@ -26,9 +26,11 @@
       }
     ],
     "distortion": {
-      "k1_red": 1.01999,
-      "k1_green": 1.248,
-      "k1_blue": 1.48601
+				"distance_scale_x": 1,
+				"distance_scale_y": 1,
+				"polynomial_coeffs_red": [ 0, 1, 0.2 ],
+				"polynomial_coeffs_green": [ 0, 1, 0.24 ],
+				"polynomial_coeffs_blue": [ 0, 1, 0.32 ]
     },
     "rendering": {
       "right_roll": 0,
```

Comment:
These changes look like they are for the better.  Kevin was not sure why they showed up in the diff because he didn't intend to make them.  This may have been my doing along the way.